src/server/helpers/github.js

```diff
@@ -11,31 +11,35 @@ import { conf } from '../helpers/config';
 const GITHUB_API_ENDPOINT = conf.get('GITHUB_API_ENDPOINT');
 const HTTP_PROXY = conf.get('HTTP_PROXY');
 
-export const requestGitHub = (options, callback) => {
-  const params = {
-    ...options,
-    proxy: HTTP_PROXY
-  };
-  params.uri = GITHUB_API_ENDPOINT + params.uri;
-  if (!params.headers) {
-    params.headers = {};
-  }
-  params.headers['User-Agent'] = 'SnapcraftBuild';
-  return request(params, callback);
+export const requestGitHub = options => {
```

Comment:
I would be for using `(options) =>` as it makes it a little bit clearer that is a function with an argument.